mobilizon_reshare/cli/cli.py

```diff
@@ -8,98 +7,156 @@
 from mobilizon_reshare.cli.commands.format.format import format_event
 from mobilizon_reshare.cli.commands.inspect.inspect_event import inspect_events
+from mobilizon_reshare.cli.commands.inspect.inspect_publication import (
+    inspect_publications,
+)
 from mobilizon_reshare.cli.commands.start.main import main as start_main
 from mobilizon_reshare.cli.commands.recap.main import main as recap_main
+from mobilizon_reshare.config.config import current_version
 from mobilizon_reshare.config.publishers import publisher_names
 
 from mobilizon_reshare.event.event import EventPublicationStatus
+from mobilizon_reshare.models.publication import PublicationStatus
 
 status_name_to_enum = {
-    "waiting": EventPublicationStatus.WAITING,
-    "completed": EventPublicationStatus.COMPLETED,
-    "failed": EventPublicationStatus.FAILED,
-    "partial": EventPublicationStatus.PARTIAL,
-    "all": None,
+    "event": {
+        "waiting": EventPublicationStatus.WAITING,
+        "completed": EventPublicationStatus.COMPLETED,
+        "failed": EventPublicationStatus.FAILED,
+        "partial": EventPublicationStatus.PARTIAL,
+        "all": None,
+    },
+    "publication": {
+        "completed": PublicationStatus.COMPLETED,
+        "failed": PublicationStatus.FAILED,
+        "all": None,
+    },
 }
 
 settings_file_option = click.option(
+    "-f",
     "--settings-file",
     type=click.Path(exists=True),
     help="The path for the settings file. "
     "Overrides the one specified in the environment variables.",
 )
 from_date_option = click.option(
+    "-b",
     "--begin",
     type=click.DateTime(),
     expose_value=True,
-    help="Include only events that begin after this datetime",
+    help="Include only events that begin after this datetime.",
 )
 to_date_option = click.option(
+    "-e",
     "--end",
     type=click.DateTime(),
     expose_value=True,
-    help="Include only events that begin before this datetime",
+    help="Include only events that begin before this datetime.",
+)
+event_status_option = click.option(
+    "-s",
+    "--status",
+    type=click.Choice(list(status_name_to_enum["event"].keys())),
+    default="all",
+    expose_value=True,
+    help="Include only events with the given STATUS.",
+)
+publication_status_option = click.option(
+    "-s",
+    "--status",
+    type=click.Choice(list(status_name_to_enum["publication"].keys())),
+    default="all",
+    expose_value=True,
+    help="Include only publications with the given STATUS.",
 )
 
 
-class InspectTarget(Enum):
-    ALL = "all"
-    WAITING = "waiting"
-
-    def __str__(self):
-        return self.value
+def print_version(ctx, param, value):
+    if not value or ctx.resilient_parsing:
+        return
+    click.echo(current_version())
+    ctx.exit()
 
 
 @click.group()
-def mobilizon_reshare():
-    pass
+@click.option(
+    "--version", is_flag=True, callback=print_version, expose_value=False, is_eager=True
+)
+@settings_file_option
```

Comment:
having the option here means the settings can only be specified before all the other commands. It's very clunky because you have to write stuff like

`mobilizon-reshare -f "..." inspect event all`

I would prefer to have it on each command.